demo/js/demo-setReturnFocus.js

```diff
@@ -0,0 +1,53 @@
+const { useState } = require('react');
+const React = require('react');
+const ReactDOM = require('react-dom');
+const FocusTrap = require('../../dist/focus-trap-react');
+
+const container = document.getElementById('demo-setReturnFocus');
+
+const focusTrapOptions = {
+  setReturnFocus: '#AlternateReturnFocusElement',
```

Comment:
> I think we need to do something similar to https://github.com/focus-trap/focus-trap/blob/master/index.js#L222 here, because we're ignoring the setReturnFocus option (as we were before; this is an existing bug).

I've added support for the `setReturnFocus` prop though the code coverage has dipped a bit since I don't think the entire `getNodeForOption` function is being used.